core/slim_sim.cpp

```diff
@@ -6435,7 +6397,7 @@ void SLiMSim::WriteProvenanceTable(tsk_table_collection_t *p_tables, bool p_use_
 	j["metadata"]["individuals"]["flags"]["17"]["name"] = "SLIM_TSK_INDIVIDUAL_REMEMBERED";
 	j["metadata"]["individuals"]["flags"]["17"]["description"] = "the individual was requested by the user to be remembered";
 	j["metadata"]["individuals"]["flags"]["18"]["name"] = "SLIM_TSK_INDIVIDUAL_FIRST_GEN";
-	j["metadata"]["individuals"]["flags"]["18"]["description"] = "the individual was in the first generation of a new population";
+	j["metadata"]["individuals"]["flags"]["18"]["description"] = "the individual was in the first generation of a new population (DEPRECATED)";
```

Comment:
Since we no longer write this flag out (right?), do we need to describe it in our metadata?